docs/contents/users/index.md

```diff
@@ -1,28 +1,19 @@
-## Create an EKS Distro Cluster
+## Using EKS Distro 
 
-The best way to create an EKS Distro cluster may be using one of our
-[Partners](../community/partners.md). There are links and instructions on that
-page for third party methods to install EKS-D.
+The best way to try out EKS Distro (EKS-D) is to use one of the Community
+procedures (such as [Starting EKS-D with kubeadm](../users/install/kubeadm-onsite.md))
```

Comment:
I think kubeadm would be the *last* way most people would want to install.

Reply:
Which would you suggest we highlight? Do we have another procedure that works outside of AWS? I think the reason I was asked to write that up is so there was a way someone could try EKS-D on-premises and not rely on the AWS cloud.

Reply:
I moved the Partners selection to the top of the left nav and changed the description to say that the Partners procedures are preferable over the Community procedures.